### PR TITLE
Maps, part one

### PR DESCRIPTION
**Pull request by kristjanjansen**
* Introduces `Dotmap` component
* Has following generator scripts:
    * Fetching and converting the dots from GeoJSON country. The data is used for rendering the `Dotmap` component
    * Fetching and converting airports database from OpenFlights. The data is not yet used anywhere
    * Fetching and converting cities metadata from Geonames database. The data is not yet used anywhere

### CODE REVIEW

File: scripts/generate.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+curl https://raw.githubusercontent.com/jpatokal/openflights/master/data/airports.dat > data/airports.csv
+
+curl $1/api/destinations > data/destinations.json
+
+curl $1/api/destinations/data > data/destinations_data.json
+
+printf "\n\e[0;32mGenerating the dots for the Dotmap component\e[0;37m"
+printf "\nThe script will take around 10 minutes to run\n\n"
+
+node generate_dots.js > ../config/dots.php
```
Comment (mikkpokk) [inline scripts/generate.sh]: Seems like generate.sh command (also tried with sudo) doesn't work as it should.

> ./scripts/generate.sh http://trip2.dev
> sudo ./scripts/generate.sh http://trip2.dev

./scripts/generate.sh: line 24: ../config/dots.php: No such file or directory

./scripts/generate.sh: line 28: ../config/airports.php: No such file or directory

./scripts/generate.sh: line 32: ../config/cities.php: No such file or directory

File: resources/views/v2/components/Dotmap/Dotmap.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    export default {
+
+        props: {
+            isclasses: { default: '' },
+            dots: { default: [] },
+            width: { default: 750 }
+        },
+
+        data: () => ({
```
Comment (mikkpokk) [inline resources/views/v2/components/Dotmap/Dotmap.vue]: Is is necessary?

`data: () ...`

Comment (kristjanjansen) [inline resources/views/v2/components/Dotmap/Dotmap.vue]: Will remove

File: scripts/generate.sh (hunk 1 above)
Comment (kristjanjansen) [inline scripts/generate.sh]: My bad, need to adjust the paths so the scripts can be run from root dir (currently assumes you are in `/scripts`

**Pull request merged**